corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -312,6 +312,7 @@ def get_all_instances_referenced_in_xpaths(app, xpaths):
         if not xpath:
             continue
 
+        xpath = xpath.replace("&quot;", '"')
```

Comment:
What would you think of `html.escape`? Every time I see specific entities replaced I wonder if the author really captured every possible relevant entity.

Reply:
I think you mean `html.unescape`. I was cautious about unescaping other characters but I guess it's probably safe.

I had hoped there was a complementary version of `xml.sax.saxutils.quoteattr` but there isn't.

Reply:
98484259a1f